docs/data-sources/process.md

```diff
@@ -0,0 +1,83 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "steampipecloud_process Data Source - terraform-provider-steampipecloud"
+description: |-
+  Use this data source to retrieve information about a process for an identity or identity workspace.
+---
+
+# Data Source: steampipecloud_process
+
+Use this data source to retrieve information about a process that belongs to an identity or identity workspace.
+
+## Example Usage
+
+**Retrieve information about a particular process id for a workspace**
+
+```terraform
+data "steampipecloud_process" "process" {
+    workspace  = "dev"
+    process_id = "p_cfbum07m1tuo1pqt843g"
+}
+```
+
+**Retrieve information about the last process for a pipeline**
+
+```terraform
+resource "steampipecloud_workspace" "test_user_workspace" {
+    handle = "%s"
+}
+
+resource "steampipecloud_workspace_pipeline" "daily_cis_pipeline" {
+    workspace_handle = steampipecloud_workspace.test_user_workspace.handle
+    title = "Daily CIS Job"
+    pipeline = "pipeline.save_snapshot"
+    frequency = jsonencode({
+      "type": "interval",
+      "schedule": "daily"
+    })
+    args = jsonencode({
+      "resource": "aws_compliance.benchmark.cis_v140",
+      "snapshot_tags": {
+        "series": "daily_cis"
+      },
+      "visibility": "workspace",
+      "inputs": {},
+      "notifications": {},
+      "variables": {}
+    })
+    tags = jsonencode({
+	  "name": "daily_cis_pipeline"
+	})
+}
+
+data "steampipecloud_process" "process_run" {
+    workspace  = steampipecloud_workspace.test_user_workspace.handle
+    process_id = steampipecloud_workspace_pipeline.daily_cis_pipeline.last_process_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `organization` - (Optional) The handle of the organization to retrieve the process for.
+- `workspace` - (Optional) The handle of the workspace to retrieve the process for.
+- `process_id` - (Required) The id of the process to be retrieved.
```

Comment:
Taking a cue from https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function#argument-reference I believe we should put the Required parameter first, followed by optional arguments in a sorted order.